### ISSUE TITLE
Accessibility issue: Heading levels should only increase by one on /

### ISSUE THREAD

**Issue by willingc** [willingc/willing-consulting-2022#14]
## What
An accessibility scan flagged the element `<h3 id="simplifying-open-source-for-forward-thinking-businesses">Simplifying open source for forward-thinking businesses</h3>` on https://willingconsulting.com because heading levels should only increase by one. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright.

To fix this, ensure the order of headings is semantically correct.

Specifically:

Fix any of the following:
-   Heading order invalid

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
